languages/python/exercises/concept/chaitanas-colossal-coaster/.docs/introduction.md

```diff
@@ -137,23 +119,21 @@ print(ex_list)
 #=> ["Tony", "Thor", "Natasha", "Bruce"]
 ```
 
-If you have a list of items that are complex, you can use the key parameter in the sort. Lets see more about this later.
+For cases where changing your original list is undesirable, the built-in [`sorted()`][sorted] can be used to return a new, sorted copy of your original list.
```

Comment:
Should we also mention `.reversed()`?

Reply:
Yes!  Will add it now.  Good catch.

Reply:
humm..._except_ `reversed()` returns a _reverse iterator_ - not a copy.  Would that be in or out of scope here?  `list.reverse()` mutates the list.